src/GHC/Util.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE PackageImports #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# OPTIONS_GHC -fno-warn-missing-fields #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Was this necessary?

Reply:
An intermediate thing. I'll now remove it.